tests/e2e/test_rosacli_cluster.go

```diff
@@ -1034,3 +1048,50 @@ var _ = Describe("Create cluster with availability zones testing",
 				Expect(common.ReplaceCommaSpaceWithComma(mp.AvalaiblityZones)).To(Equal(availabilityZones))
 			})
 	})
+
+var _ = Describe("Create cluster with fips testing",
```

Comment:
No need another Describe here. It's day1post  put it together with other day1-post cases is fine.

Reply:
remove day1-post with other day1-post case together.